tutorial/bug_anim/assets/sprites/Fighter/index.json

```diff
@@ -0,0 +1,226 @@
+            "path": "__animation_walk_walk-2.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-3",
+            "path": "__animation_walk_walk-3.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-4",
+            "path": "__animation_walk_walk-4.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-5",
+            "path": "__animation_walk_walk-5.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-6",
+            "path": "__animation_walk_walk-6.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-7",
+            "path": "__animation_walk_walk-7.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_walk_walk-8",
+            "path": "__animation_walk_walk-8.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-1",
+            "path": "__animation_default_idle-1.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-2",
+            "path": "__animation_default_idle-2.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-3",
+            "path": "__animation_default_idle-3.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-4",
+            "path": "__animation_default_idle-4.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-5",
+            "path": "__animation_default_idle-5.png"
+        },
+        {
+            "x": 0,
+            "y": 0,
+            "faceRight": 0,
+            "bitmapResolution": 2,
+            "name": "__animation_default_idle-6",
+            "path": "__animation_default_idle-6.png"
+        }
+    ],
+    "fAnimations": {
+        "fight": {
+            "from": "__animation_fight_attack_1-1",
+            "to": "__animation_fight_attack_1-4",
```

Comment:
I just tried setting `frameFrom` & `frameTo` instead of `from` & `to`

```suggestion
            "frameFrom": "__animation_fight_attack_1-1",
            "frameTo": "__animation_fight_attack_1-4",
```

it panics:

```
panic: interface conversion: interface {} is nil, not string

goroutine 1 [running, locked to thread]:
github.com/goplus/spx.(*Sprite).init(0x140001e5e60, {0x14000325938?, 0x100dc3c08?}, 0x140000e4280, {0x1400052b510, 0x7}, 0x140001a3290, {0x100d8e6a0?, 0x140000e4280?, 0x10078e9f4?}, ...)
        /Users/nighca/code/goplus/spx/sprite.go:154 +0x904
github.com/goplus/spx.(*Game).loadSprite(0x140000e4280, {0x100dc53a0?, 0x140001e5e60}, {0x1400052b510, 0x7}, {0x100d8e6a0?, 0x140000e4280?, 0x10078e9f4?})
        /Users/nighca/code/goplus/spx/game.go:369 +0x2f4
github.com/goplus/spx.(*Game).newSpriteAndLoad(0x100d265c0?, {0x1400052b510, 0x7}, {0x100dcc848?, 0x100db98a0?}, {0x100d8e6a0?, 0x140000e4280?, 0x1011aea40?})
        /Users/nighca/code/goplus/spx/game.go:131 +0x94
github.com/goplus/spx.(*Game).getSpriteProtoByName(0x140000e4280, {0x1400052b510, 0x7}, {0x100d8e6a0?, 0x140000e4280?, 0x1007dfbd0?})
        /Users/nighca/code/goplus/spx/game.go:154 +0xfc
github.com/goplus/spx.(*Game).loadIndex(0x140000e4280, {0x100d8e6a0?, 0x140000e4280?, 0x100cb978b?}, 0x140001a31e0)
        /Users/nighca/code/goplus/spx/game.go:413 +0x3bc
github.com/goplus/spx.(*Game).endLoad(0x100d8e6a0?, {0x100d8e6a0?, 0x140000e4280?, 0x140004dfe38?}, 0x100beeb76?)
        /Users/nighca/code/goplus/spx/game.go:460 +0xb8
github.com/goplus/spx.Gopt_Game_Run({0x100dc1d28?, 0x140000e4280}, {0x100cfbc20?, 0x100dbfb28?}, {0x0, 0x0, 0x140002d9f01?})
        /Users/nighca/code/goplus/spx/game.go:269 +0x594
github.com/goplus/spx.Gopt_Game_Main({0x100dc1d28, 0x140000e4280}, {0x14000021160?, 0x0?, 0x140002d9f28?})
        /Users/nighca/code/goplus/spx/game.go:187 +0x98
main.(*Game).Main(...)
        /Users/nighca/code/goplus/spx/tutorial/bug_anim_nighca/gop_autogen.go:20
main.main()
        tutorial/bug_anim_nighca/Fighter.spx:29 +0x90
exit status 2
exit status 1
```